clang/lib/Driver/ToolChains/CommonArgs.cpp

```diff
@@ -490,6 +490,17 @@ void tools::AddLinkerInputs(const ToolChain &TC, const InputInfoList &Inputs,
     else
       A.renderAsInput(Args, CmdArgs);
   }
+  if (const Arg *A = Args.getLastArg(options::OPT_fveclib)) {
+    if (A->getNumValues() == 1) {
```

Comment:
fveclib is a Joined option that takes exactly one value. `getNumValues() == 1` is an unneeded test

Reply:
ok